dask_jobqueue/tests/__init__.py

```diff
@@ -1,3 +1,3 @@
 from __future__ import absolute_import, division, print_function
 
-QUEUE_WAIT = 15  # seconds
+QUEUE_WAIT = 120  # seconds
```

Comment:
120 seconds is a lot of time!

Reply:
Yeah, the scheduler I could test with has longer waiting times...
I run my tests with a longer QUEUE_WAIT, and I forgot to revert to the default value before submitting the pull request.

Should be fixed now.